backtesting/backtesting.py

```diff
@@ -468,6 +473,14 @@ def tp(self) -> Optional[float]:
     def parent_trade(self):
         return self.__parent_trade
 
+    @property
+    def id(self) -> Optional:
+        """
+        An ID attribute which, if set, persists to enable tracking of this order
+        if it becomes a trade in `Strategy.trades` and when closed in `Strategy.closed_trades`.
```

Comment:
Maybe:

> An arbitrary tag object which, if set, propagates to the `Trade` this order started.

Which begs the question, what about orders that _close_ existing trades?